### ISSUE TITLE
Terms in PMID25869666 regarding karyogamy

### ISSUE THREAD

**Issue by manulera** [pombase/fypo#4139]
(still incomplete, not worth reading)

To be discussed in a meeting

> https://www.pombase.org/term/FYPO:0000513

Following work on https://github.com/manulera/discussions_curation/issues/7

## Generic comments

* Decreased nuclear fusion during mating (FYPO:0000512). Shouldn't this be abolished nuclear fusion (FYPO:0000511)  with a certain penetrance?
* what is conjugation in yeast?

## Specific terms to differentiate nuclear congression / karyogamy / horsetail

I think "nuclear migration during mating" (see below) can be ambiguous, as it could be interpreted both as horsetail or nuclear congression (the process in which, after cells have fused, nuclei come together).

Also nuclear congression and karyogamy are not exactly the same. Congression is required for karyogamy (nuclei have to come together to fuse), but you can have nuclei coming together without fusing once they have made contact.

* Decreased nuclear migration during mating (FYPO:0006129). I think this term should be more specific, indicating whether it refers to nuclear congresion or horsetail.
  * This term has only one annotation in which ssm4D is shown to alter horsetail. Not entirely sure to what exact phenotype it corresponds. In the paper they show that ssm4D inhibits horsetail movement, but this has a different annotation. Also the annotation mentions a penetrance of 20% that does not seem to relate to the percentages from the paper. The paper does not mention nuclear congresion.
* Same applies to its parent FYPO:0003062 (abnormal nuclear migration during mating).
  * The extra annotation from mal3 refers to nuclear congression.

**Comment by manulera** [comment]
Done!

**Comment by manulera** [comment]
> Decreased nuclear migration during mating (FYPO:0006129). I think this term should be more specific, indicating whether it refers to nuclear congresion or horsetail.

I see that 

FYPO:0000512 | decreased nuclear fusion during mating | dhc1Δ
-- | -- | --

is still there. was this change made and it has not been updated yet, or we did not do it?

**Comment by manulera** [comment]
I would also need this new term

| ID | Term name                                   | Parent       | Reason        | Definition                                                                                                                                                       |
| -- | ------------------------------------------- | ------------ | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| 1  | Abolished nuclear congression during mating | FYPO:0000511 | more specific | A cellular process phenotype in which karyogamy involved in conjugation with cellular fusion does not occur because the nuclei fail to come in physical contact. |

Then, I will move this annotation with 40% penetrance to this new term:

mal3Δ | FYPO:0003062 | abnormal nuclear migration during mating | Microscopy |   | Yamashita A et al. (2013)
-- | -- | -- | -- | -- | --

This action would leave these two terms orphaned, which should be deleted:
* FYPO:0003062
* FYPO:0006129

**Comment by ValWood** [comment]
https://github.com/pombase/fypo/issues/4146
i didn't do it yet. I need to check how to do obsoletions but I will do the changes and run the pipeline on Monday (It tasks a while and the terms do not appear in Canto  until the pipeline is run and a new release  created).

But I was going to ask because I thought we did move the annotations already but I still see them...

**Comment by manulera** [comment]
Some extra terms

| ID | Term name                                                               | Parent       | Reason                                                   | Definition                                                                                                                                                         |  |
| -- | ----------------------------------------------------------------------- | ------------ | -------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |  |
| 1  | abolished protein localization to microtubule plus-end during karyogamy | FYPO:0005989 | Plus end localization only defined for vegetative growth | A cell phenotype observed during nuclear congression involved in karyogamy in which the localization of a protein to the plus ends of microtubules does not occur. |  |
| 2  | abolished protein localization to spindle pole body during karyogamy    | FYPO:0003542 | More specific                                            | A cell phenotype in which the localization of a protein to the spindle pole body is abolished during karyogamy.                                                    |  |

**Comment by manulera** [comment]
Regarding the question:

* FYPO:0005989 -> abnormal protein localization during mating
* FYPO:0006130 -> abolished protein localization to microtubule during meiosis

I imagine that the right parent is FYPO:0006130, since mating probably refers to the period before cell fusion and once karyogamy starts it is already meiosis?

But then, FYPO:0005989 has this child [FYPO:0005990](https://www.pombase.org/term/FYPO:0005990) (abnormal maintenance of protein location in cell cortex of cell tip during karyogamy)

In summary, if these proteins are mislocalized during karyogamy, I am not sure what parents they should have.

**Comment by manulera** [comment]
Added the new terms. TODO:

* [ ] Move annotation of mal3 from FYPO:0003062 to FYPO:0007989 (new term added)
* [ ] Obsolete FYPO:0006129
